src/components/Branding/Logo/Logo.tsx

```diff
@@ -0,0 +1,33 @@
+import { SvgIcon } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+
+import { LogoLarge, LogoMaximized, LogoSmall } from '../';
+
+interface Props {
+  variant?: string;
+}
+
+const Logo: React.FC<Props> = (svgProps) => {
+  const theme = useTheme();
+  const color = theme.palette.mode === 'dark' ? theme.palette.common.white : '#323232';
```

Comment:
Back to this :) .  If this works I don't want to be asking you to change it anymore, but consider this.  In `theme.ts`, you can extend the `interface Palette` extension to include `logo: string;` and the `PaletteOptions` to include `logo?: string;`... from there, you can define the logo color on each theme as a just a string the same way `divider` is defined.  On `dark` I presume it would be `#FFF` and on light it would be `#323232`.   These declarations turn into `const color = theme.plalette.logo`.   The color is fully managed in the `palette.ts` file and we don't have the misc logic / 'magic strings' in these files.